app/assets/stylesheets/finder_frontend.scss

```diff
@@ -48,7 +48,7 @@
     dt {
       display: none;
 
-      &.metadata-date-label {
+      &.metadata-date-label, &.metadata-text-label {
```

Comment:
If `metadata-date-label` and `metadata-text-label` have exactly the same styles, can I suggest merging them into one `metadata-label` (perhaps) ?